### PR TITLE
Update torch version 1.3.1 to 1.6.0

### PR DESCRIPTION
**Pull request by jiseongHAN**
#268  

In R2D1 brain.py specify tensor type to use .float().
In PPO, seperate loss that reapeated in training phase.
In SAC, Train actor loss first and then, Q, V to be trained.

**lunarlander_continuous**
a2cddpgtd3sacppobcddpgsacfdddpgfdbcsac

A2C / DDPG / TD3 / SAC / PPO / BC_DDPG / SACfD / DDPGfD / BC_SAC

### CODE REVIEW

File: rl_algorithms/bc/sac_learner.py [hunk 1]
```diff
@@ -95,6 +95,7 @@ def update_model(
         actor_loss = torch.zeros(1)
         n_qf_mask = 0
         if self.update_step % self.hyper_params.policy_update_freq == 0:
+            q_pred = torch.min(self.qf_1(states_actions), self.qf_2(states_actions))
```
Comment (MrSyee) [inline rl_algorithms/bc/sac_learner.py:98]: You should consider to move `qf_1` update part down.

**Review by jinPrelude — APPROVED** [review]
